docs/application/toc_all.md

```diff
@@ -979,9 +979,9 @@
 #### [Tracepoints](/application/native/guides/performance/tracepoints.md)
 
 ### Watchface Complication
-#### [Overview](/application/native/guides/compliations/overview.md)
-#### [Watchface Complication](/application/native/guides/compliations/watchface-complication.md)
-#### [Complication Provider](/application/native/guides/compliations/watchface-complication-provider.md)
+#### [Overview](/application/native/guides/complications/overview.md)
+#### [Watchface Complication](/application/native/guides/complications/watchface-complication.md)
+#### [Complication Provider](/application/native/guides/complications/watchface-complication-provider.md)
```

Comment:
I cam see the same lines are located at the wrong place too, from line 498 to line 501.

Would you please delete them also?

![image](https://user-images.githubusercontent.com/9958374/74810688-4720f080-5333-11ea-8ca2-ebc7b360206a.png)